app/controllers/application_controller.rb

```diff
@@ -11,23 +14,25 @@ class ApplicationController < Sinatra::Application
     enable :sessions
     set :session_secret, ENV.fetch('SESSION_SECRET') { SecureRandom.hex(64) }
     # Step 2: Enable Rack::Protection by uncommenting the following lines:
-    #
-    # use Rack::Protection
-    # # `use Rack::Protection` automatically enables all modules except for the
-    # # following, which have to be enabled explicitly
-    # use Rack::Protection::AuthenticityToken
+    use Rack::Protection
+    use Rack::Protection::AuthenticityToken
+    # removed to allow for escaping of HTML tags
     # use Rack::Protection::EscapedParams
```

Comment:
`Rack::Protection::EscapedParams` is missing here.  It provides more robust protection than just escaping text as we insert it into the HTML because it prevents us from storing malicious data

Reply:
I had to remove that to allow for one of the tests to pass (the one with the HTML tags in it). I thought we hadn't found any other way to fix that at the time I submitted this